src/style.css

```diff
@@ -0,0 +1,109 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.first_form {
+  margin: 20px 0;
+}
+
+.input__wrapper__big {
+  position: relative;
+}
+
+.input__wrapper__big img {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  top: 50%;
+  transform: translateY(-50%);
+  left: 26px;
+}
+
+.large {
+  width: 100%;
+  height: 70px;
+  top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  padding: 25px 62px;
+  font-family: inherit;
+  font-size: 16px;
+  box-shadow: 1px 1px 5px rgb(0 0 0 / 10%);
+}
```

Comment:
here you duplicate a lot of styles in `large` and `small`. You need to rewrite classes as I have mentioned before to resolve it.